chingu-coders/Voyage2-Turtles-02
# make user-defined newssources persistent

## Purpose
* Make user's newsSources choices persist across reloads

## Approach/Fixes
* MAJOR OVERSIGHT on my part. kinda the whole point of letting the user define the sources

## Testing
* Check out this PR
* modify the news sources
* reload and confirm that they persist
